htdocs/web_portal/controllers/ngi/view_ngi.php

```diff
@@ -36,15 +36,9 @@ function view_ngi() {
 
     $params['portalIsReadOnly'] = portalIsReadOnlyAndUserIsNotAdmin($user);
 
-    $params['UserIsAdmin']=false;
-    if(!is_null($user)) {
-        $params['UserIsAdmin']=$user->isAdmin();
-    }
-
-    $params['authenticated'] = false;
-    if($user != null){
-        $params['authenticated'] = true;
-    }
+    // Set values for showing personal data
+    $params = array();
```

Comment:
Does this "reset" `$params` to an empty array?

Reply:
Good spot, thanks.